### PR TITLE
ABB-717 Add random resource name back to starterkit

### PR DESCRIPTION
**Pull request by Alramech**
Uses timestamp to regenerate the random_pet

Tested on prod

### CODE REVIEW

**Comment by Alramech** [comment]
Current dependencies on/for this PR:
* main
  * **PR #5** <a href="https://app.graphite.dev/github/pr/abbeylabs/abbey-starter-kit-quickstart/5" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/abbeylabs/abbey-starter-kit-quickstart/5?utm_source=stack-comment).

File: main.tf [hunk 1]
```diff
@@ -45,11 +54,18 @@ resource "abbey_grant_kit" "abbey_demo_site" {
     # Path is an RFC 3986 URI, such as `github://{organization}/{repo}/path/to/file.tf`.
     location = "github://replace-me-with-organization/replace-me-with-repo/access.tf" # CHANGEME
     append = <<-EOT
-      resource "abbey_demo" "grant_read_write_access" {
+      resource "abbey_demo" "grant_read_write_access_${random_pet.random_pet_name.id}" {
```
Comment (songe) [inline main.tf:57]: What does a sample output look like?

Comment (Alramech) [reply main.tf:57]: Example 
image

**Review by songe — APPROVED** [review]
:shipit:

**Comment by Alramech** [comment]
Archiving for now until we get a proper method of random name generaion

**Pull request closed without merging**